content/kubermatic/main/tutorials-howtos/gitops-argocd/_index.en.md

```diff
@@ -0,0 +1,241 @@
+The seed need it's own DNS prefix which configured as `self.seed`. This prefix needs to be configured in Route53 or similar DNS provider in your setup.
+
+Similarly, the demo creates 2nd seed named `india-seed`. Thus, 2nd seed's argocd, prometheus, grafana etc are accessible at `*.india.argodemo.lab.kubermatic.io`. And this seed's DNS prefix is `india.seed`.
+
+These names would come handy to understand below references to them and customize these values as per your setup.
+
+### Installation of KKP Master-seed combo
+1. Install ArgoCD and all the ArgoCD Apps
+    ```shell
+    cd <root directory of this repo>
+    make deploy-argo-dev-master deploy-argo-apps-dev-master
+    # get argocd admin password via below command
+    kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
+    ```
+1. Tag the git with right label. The make target creates a git tag with a pre-configured name: `dev-kkp-<kkp-version>` and pushes it to your git repository. This way, when you want to upgrade KKP version, you just need to update the KKP version at the top of Makefile and run this make target again.
+    ```shell
+    make push-git-tag-dev
+    ```
+1. ArgoCD syncs nginx ingress and cert-manager automatically
+1. Manually update the DNS records so that ArgoCD is accessible.
+    ```shell
+    # Apply below DNS CNAME record manually in AWS Route53 for below:
+    #   argodemo.lab.kubermatic.io
+    #   *.argodemo.lab.kubermatic.io
+    #   grafana-user.self.seed.argodemo.lab.kubermatic.io
+    #   alertmanager-user.self.seed.argodemo.lab.kubermatic.io
+    # You can get load balancer details from `k get svc -n nginx-ingress-controller nginx-ingress-controller`
+    # After DNS setup, you can access ArgoCD at https://argocd.argodemo.lab.kubermatic.io
+    ```
+1. Install KKP EE without helm-charts. If you would want complete ArgoCD setup with separate seeds, we will need Enterprise Edition of KKP. You can run the demo with master-seed combo. For this,  community edition of KKP is sufficient.
+    ```shell
+    make install-kkp-dev
+    ```
+1. Apply ClusterIssuer. **Note:** For the demo, staging LetsEncrypt certificate provider is used. In real world production applications, you should use production LetsEncrypt certificate issuer. Just need to change the url in `dev/clusterIssuer.yaml` file.
+    ```shell
+    kubectl apply -f dev/clusterIssuer.yaml
+    ```
+1. Add seed for self (need manual update of kubeconfig in seed.yaml)
+    ```shell
+    make create-long-lived-master-seed-kubeconfig
+    # commit changes to git and push latest changes in
+    make push-git-tag-dev
+    ```
+1. Sync all apps in ArgoCD by accessing ArgoCD UI and syncing apps manually
+1. Seed DNS record AFTER seed has been added (needed for usercluster creation). Seed is added as part of ArgoCD apps reconciliation above
```

Comment:
"Seed DNS record AFTER seed has been added (needed for usercluster creation)" is missing a verb.